app/src/main/java/org/fossasia/openevent/app/data/models/Attendee.java

```diff
@@ -179,4 +200,171 @@ public Attendee createFromParcel(Parcel source) {
             return new Attendee[size];
         }
     };
+
+    @Override
+    public int compareTo(@NonNull Attendee other) {
+        int compareFirstName;
+        int compareLastName;
+        return (compareFirstName = firstName.toLowerCase().compareTo(other.getFirstName().toLowerCase())) == 0 ? (compareLastName = lastName.toLowerCase().compareTo(other.getLastName().toLowerCase())) == 0 ? email.toLowerCase().compareTo(other.getEmail().toLowerCase()) : compareLastName : compareFirstName;
+    }
+
+    @Override
+    public Object getTag() {
+        return null;
+    }
+
+    @Override
+    public Attendee withTag(Object o) {
+        return null;
+    }
+
+    @Override
+    public boolean isEnabled() {
+        return false;
+    }
+
+    @Override
+    public Attendee withEnabled(boolean b) {
+        return null;
+    }
+
+    @Override
+    public boolean isSelected() {
+        return false;
+    }
+
+    @Override
+    public Attendee withSetSelected(boolean b) {
+        return null;
+    }
+
+    @Override
+    public boolean isSelectable() {
+        return false;
+    }
+
+    @Override
+    public Attendee withSelectable(boolean b) {
+        return null;
+    }
+
+    @Override
+    public int getType() {
+        return 0;
+    }
+
+    @Override
+    public int getLayoutRes() {
+        return R.layout.attendee_layout;
+    }
+
+    @Override
+    public View generateView(Context context) {
+        ViewHolder holder = getViewHolder((ViewGroup) LayoutInflater.from(context).inflate(getLayoutRes(), null, false));
+
+        bindView(holder, Collections.EMPTY_LIST);
+
+        return holder.itemView;
+    }
+
+    @Override
+    public View generateView(Context context, ViewGroup viewGroup) {
+        ViewHolder holder = getViewHolder(viewGroup);
+
+        bindView(holder, Collections.EMPTY_LIST);
+
+        return holder.itemView;
+    }
+
+    @Override
+    public ViewHolder getViewHolder(ViewGroup viewGroup) {
+        return new ViewHolder(AttendeeLayoutBinding.inflate(LayoutInflater.from(viewGroup.getContext()), viewGroup, false));
+    }
+
+    @Override
+    public void bindView(ViewHolder viewHolder, List<Object> list) {
+        viewHolder.bindAttendee(this);
+    }
+
+    @Override
+    public void unbindView(ViewHolder viewHolder) {
+        viewHolder.unbindAttendee();
+    }
+
+    @Override
+    public void attachToWindow(ViewHolder viewHolder) {
+
+    }
+
+    @Override
+    public void detachFromWindow(ViewHolder viewHolder) {
+
+    }
+
+    @Override
+    public boolean failedToRecycle(ViewHolder viewHolder) {
+        return false;
+    }
+
+    @Override
+    public boolean equals(int i) {
+        return id == i;
+    }
+
+    @Override
+    public Attendee withIdentifier(long l) {
+        id = l;
+        return this;
+    }
+
+    @Override
+    public long getIdentifier() {
+        return id;
+    }
+
+    public static class ViewHolder extends RecyclerView.ViewHolder {
```

Comment:
That's not true. It is a generic parameter. You can move the class anywhere and just pass the type and it'll work :+1: